src/SprykerEco/Yves/Computop/ComputopFactory.php

```diff
@@ -512,6 +552,22 @@ protected function createOrderEasyCreditMapper()
         );
     }
 
+    /**
+     * @return \SprykerEco\Yves\Computop\Mapper\Init\PostPlace\PayUCeeSingleMapper
+     */
+    protected function createOrderPayUCeeSingleMapper(): PayUCeeSingleMapper
+    {
+        return new PayUCeeSingleMapper(
```

Comment:
* Rename the class to `PayuCeeSingleMapper()`.

Reply:
Done